kong/plugins/grpc-gateway/handler.lua

```diff
@@ -120,7 +120,7 @@ function grpc_gateway:body_filter(conf)
   if not ret or #ret == 0 then
     if ngx_arg[2] then
       -- it's eof and we still cannot decode, fall through
-      ret = deco:get_raw_downstream_body()
```

Comment:
It's the decoder instance dec should be used there because it's dec who carry the downstream_body.
Using deco will make get_raw_downstream_body return nil, hence the response body will be nil.